lib/events/events.proto

```diff
@@ -143,6 +143,9 @@ message SessionStart {
 
     // InitialCommand is the command used to start this session.
     repeated string InitialCommand = 9 [ (gogoproto.jsontag) = "initial_command,omitempty" ];
+
+    // AccessRequests are the IDs of access requests created by the user
+    repeated string AccessRequests = 10 [ (gogoproto.jsontag) = "access_requests,omitempty" ];
```

Comment:
IIRC the field number was `11` for this on master.  The field numbers must be consistent (its okay to skip `10`).  These numbers are used as as the "keys" in the mapping that gets sent over the wire, so having different key numbers in different versions breaks compatibility.